bin/prover-client/src/args.rs

```diff
@@ -41,6 +51,36 @@ pub struct Args {
     #[argh(option, description = "bitcoind RPC password")]
     pub bitcoind_password: String,
 
+    #[cfg(feature = "risc0")]
+    #[argh(
+        option,
+        description = "number of risc0 prover workers to spawn",
+        default = "20"
+    )]
+    pub risc0_workers: usize,
+
+    #[cfg(feature = "sp1")]
+    #[argh(
+        option,
+        description = "number of sp1 prover workers to spawn",
+        default = "20"
+    )]
+    pub sp1_workers: usize,
+
+    #[argh(
+        option,
+        description = "number of native prover workers to spawn",
+        default = "20"
+    )]
```

Comment:
Damn 20?!
Whats the rationale for that?
This is like 20 threads no?

Reply:
Yes. 
We are producing EVM EE & CL STF blocks every ~5s. The proof generation for that was taking roughly 40s in SP1. In 40s we have: 8 EVM EE block + 8 CL STF block -> 16 proofs.
In 40s we also have: 2 BTC block: total 2 proofs.

If we needed to catch up we need to have at least 18 workers.

Reply:
Can you add a comment on why 20 in the file?